raiden/transfer/state_change.py

```diff
@@ -633,23 +633,6 @@ def from_dict(cls, data: Dict[str, Any]) -> "ContractReceiveChannelSettled":
         )
 
 
-class ActionLeaveAllNetworks(StateChange):
```

Comment:
I realize that this is dead-code (for now)... shouldn't we keep this and just make sure that it's tested? the rationale would be that we might end up exposing this through the APIs, similar to ActionCancelPayment and ActionSetChannelFee.

Reply:
Why keep it if it's not used? When it will be needed in the future you can always bring it back.